lib/routes/publico/items-processor.ts

```diff
@@ -0,0 +1,19 @@
+export default function getItems(data) {
+    const items = data('.category-list li')
+        .toArray()
+        .map((item) => {
+            item = data(item);
+            const title = item.find('h2').text();
+            const link = item.find('a').attr('href');
+            const author = item.find('p').text();
+            const image = item.find('picture img').attr('src');
+
+            return {
+                title,
+                link,
+                description: `<img src="${image}" alt="${title}"><p>${author}</p>`,
```

Comment:
author should be placed inside the [`author` property](https://github.com/DIYgod/RSSHub/blob/4fa33716c103e9aa9fd5df7c74a69b5467924364/lib/types.ts#L38).